tests/posix/xsi_realtime_threads/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_POSIX_API=y
+CONFIG_ZTEST=y
+
+CONFIG_POSIX_AEP_CHOICE_BASE=y
```

Comment:
We definitely need the `CONFIG_XSI_REALTIME_THREADS` Option Group here.

It needs to be added first, as part of this ticket.
https://github.com/zephyrproject-rtos/zephyr/issues/86061